spec/registry.md

```diff
@@ -826,3 +812,24 @@ For more information, see [Working with Timezones](https://w3c.github.io/timezon
 > The form of these serializations is known and is a de facto standard.
 > Support for these extensions is expected to be required in the post-tech preview.
 > See: https://datatracker.ietf.org/doc/draft-ietf-sedate-datetime-extended/
+
+### Date and Time Locale Options
```

Comment:
Having thought about it, I think this is the wrong approach. The `MAY` means that these are optional options. We should just put them into the list for each function as **optional**

Doing so would solve the other problem I have, which is that `timeZone` is most definitely not part of locale (even if the `-u` extension provides a mechanism for transmitting the zone).